test/plugins/features/movies/controller.test.js

```diff
@@ -90,5 +90,63 @@ describe('movie controller', () => {
 
   });
 
+  describe('add', () => {
+
+    const movie_list = [
+      {
+        title: 'Argo',
+        release_year: 2012
+      },
+      {
+        title: 'Guardians of the Galaxy',
+        release_year: 2014
+      },
+      {
+        title: 'Bedazzled',
+        release_year: 2007
+      }
+    ];
+
+    const locations_list = [
+      {
+        location: 'San Francisco'
+      },
+      {
+        location: 'New York'
+      }
+    ];
+
+    beforeEach(() => {
+      return Knex.raw('TRUNCATE movies CASCADE')
+        .then(() => {
+          return Knex('movies').insert(movie_list);
+        });
+    });
+
+    beforeEach(() => {
+      return Knex.raw('TRUNCATE locations CASCADE')
+        .then(() => {
+          return Knex('locations').insert(locations_list);
+        });
+    });
+
+    it('location to movie', () => {
+      return new Movie({ title: 'Bedazzled' }).fetch()
+      .then((movie) => {
+        const payload = { location: 'New York' };
```

Comment:
Yeah i was doing this since i hadn't really provided a GET endpoint for /locations and there wasn't any way for the client to know the ID of the location. So i was just passing in the location string itself. I will make the change to pass in ID instead and also provide a GET /locations endpoint so its clear what ID to use for a given location. Thanks!